saleor/plugins/base_plugin.py

```diff
@@ -91,6 +91,10 @@ def __init__(
     def __str__(self):
         return self.PLUGIN_NAME
 
+    def check_plugin_id(self, plugin_id: str) -> bool:
```

Comment:
Don't you think that it would be good to convert this method to `classmethod`?  Sometimes we would like to check plugin_Id without creating a plugin  instance